src/protocols/vnc/display.c

```diff
@@ -51,16 +51,10 @@ void guac_vnc_update(rfbClient* client, int x, int y, int w, int h) {
     guac_client* gc = rfbClientGetClientData(client, GUAC_VNC_CLIENT_KEY);
     guac_vnc_client* vnc_client = (guac_vnc_client*) gc->data;
 
-#ifdef LIBVNC_CLIENT_HAS_SCREEN
-    int new_height = rfbClientSwap16IfLE(client->screen.height);
-    int new_width  = rfbClientSwap16IfLE(client->screen.width);
-#else
-    int new_height = rfbClientSwap16IfLE(client->height);
```

Comment:
This was wrong anyway - the bit swap should not happen on the client height and width.